HephaestusConfigurator/MainWindow.xaml

```diff
@@ -0,0 +1,91 @@
+﻿<controls:MetroWindow x:Class="HephaestusConfigurator.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:controls="clr-namespace:MahApps.Metro.Controls;assembly=MahApps.Metro"
+        WindowTitleBrush="#561D25"
+        NonActiveWindowTitleBrush="#561D25"
+        BorderBrush="#561D25"
+        NonActiveBorderBrush="#561D25"
+        Title="Hephaestus Configurator" Height="450" Width="650">
+    
```

Comment:
Maby set this to launch in the center of the screen and not the corner if haven't done so already somewhere else

Reply:
It should start in the middle of the screen anyways.